### ISSUE TITLE
Usersnap Feedback - It might be good to delete these headers for cases that have no content fo[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#834]
**Sender**: amberfj@gmail.com
**Comment**: It might be good to delete these headers for cases that have no content for these headers.
[Open #531 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/531)

<a href='https://usersnappublic.s3.amazonaws.com/2019-08-23/01-05/a2d78716-afb0-4d32-a2a8-ca474975af6a.png'>![Usersnap Feedback - It might be good to delete these headers for cases that have no content fo(...)](https://usersnappublic.s3.amazonaws.com/2019-08-23/01-05/efe838bb-b1fd-43ac-a0a6-675a25b609eb.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-08-23/01-05/a2d78716-afb0-4d32-a2a8-ca474975af6a.png'>Download original image</a>

**Browser**: Chrome 75 (macOS Sierra)
**Referer**: [https://participedia.net/case/6036](https://participedia.net/case/6036)
**Screen size**: 2560 x 1440 **Browser size**: 1512 x 1237

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
I think the editing team wants to keep them, as they may get filled out by others.

**Comment by plscully** [comment]
Yes, they are stubs for now, but part of the plan is to try and get students or other users to complete them

**Comment by ascott** [comment]
We have discussed a solution for this where we break the narrative field into multiple fields, one for each of these headings. Then when one of these fields is not filled out, it will not be displayed on the reader view, but each heading/section will be available in the edit view to be completed over time. I think we should prioritize this change as it does look pretty weird on the reader view, and it's possible for editors to just go in and delete the headings if they want to.

**Comment by plscully** [comment]
OK - this makes good sense. I like that the headings are still available on edit view. It's the best of both worlds!

**Comment by jesicarson** [comment]
@ascott I proposed that design way back, but I can't recall why I was overruled (maybe because when you export a case then the narrative would be broken up into fields). Anyway, while I don't think this change needs to be prioritized right now, I would be on board with it in the future, particularly if we hear any feedback from readers about the blank headers.
